code/lib/test/src/index.ts

```diff
@@ -1,18 +1,11 @@
 import { instrument } from '@storybook/instrumenter';
-import * as spy from '@vitest/spy';
+import { type LoaderFunction } from '@storybook/csf';
 import chai from 'chai';
-import { FORCE_REMOUNT, STORY_RENDER_PHASE_CHANGED } from '@storybook/core-events';
-import { addons } from '@storybook/preview-api';
+import { global } from '@storybook/global';
```

Comment:
you should add this as a dependency in the package

Reply:
Check